File: Codechef/CLEANUP.py
# your code goes here
t = int(input())
for _ in range(t):
	(n,m) = map(int,input().split())
	m1 = list(map(int,input().split()))
	li1 = [0]*n
	li2 = []
	chef = []
	ass = []
	m1.sort()
	for i in range(len(m1)):
		li1[m1[i]] = 1
	for i in range(1,n+1):
		if(li1[i]==0):
			li2.append(i)
	for i in range(len(li2)):
		if(i%2==0):
			chef.append(i)
		else:
			ass.append(i)
	for i in range(len(chef)):
		print(chef[i],end=" ")
	for i in range(len(ass)):
		print(ass[i],end=" ")

File: Codechef/CLEANUP.cpp
#include<bits/stdc++.h>
using namespace std;
int main(){
    int t;
    cin >> t;
    while(t--){
        int n,m;
        cin >> n >> m;
        int a[1000];
        for(int i=0;i<m;i++){
            cin >> a[i];
        }
        sort(a,a+m);
        int l[n];
        for(int i=0;i<n;i++){
            l[i]=0;
        }
        for(int i=0;i<m;i++){
            l[a[i]-1]=1;
        }
        // for(int i=0;i<n;i++){
        //     cout << l[i];
        // }
        vector<int> v;
        for(int i=0;i<n;i++){
            if(l[i]==0){
                v.push_back(i);
            }
        }
        vector<int> chef;
        vector<int> ass;
        for(int i=0;i<v.size();i++){
            if(i%2==0){
                chef.push_back(v[i]+1);
            }else
            {
                ass.push_back(v[i]+1);
            }
            
        }
        for(int i=0;i<chef.size();i++){
            cout << chef[i] << " ";
        }
        cout << endl;
        for(int i=0;i<ass.size();i++){
            cout << ass[i] << " ";
        }
    }
    return 0;
}